zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/handlers/FlowNodeInstanceFromProcessInstanceHandler.java

```diff
@@ -88,11 +88,11 @@ public void updateEntity(
     entity.setBpmnProcessId(recordValue.getBpmnProcessId());
     entity.setTenantId(tenantOrDefault(recordValue.getTenantId()));
 
-    // Parent tree path is intentionally not calculated here, we set the value as if the instance is
-    // on the 1st level
-    // will be changed within https://github.com/camunda/camunda/issues/18378
-    entity.setTreePath(recordValue.getProcessInstanceKey() + "/" + record.getKey());
-    entity.setLevel(1);
+    if (intentStr.equals(ELEMENT_ACTIVATING.name())) {
+      // we set the default value here, which may be updated later within incident export
+      entity.setTreePath(recordValue.getProcessInstanceKey() + "/" + record.getKey());
+      entity.setLevel(1);
+    }
```

Comment:
👍🏼 So we always have at least a flat treePath.

This is only true as long flowscope is same as PI right? (or if flowscope is -1, I guess)

Reply:
Yes, this tree path will be a "real" tree path only for cases when flowScope = -1. I'm not sure whether it's fine to leave it here. Will check in my further testing.